src/theme/index.tsx

```diff
@@ -385,6 +385,7 @@ body {
 
 .custom-toast-root {
     margin-top: 86px;
+    width:350px !important;
```

Comment:
Is `!important` necessary here?

Reply:
the whole line doesn't seem important for this feature.
Removing.

Reply:
![Screenshot 2022-06-06 at 20 20 28](https://user-images.githubusercontent.com/102727631/172222005-77a02bdf-7cfa-43f4-a31d-15182d8f857f.png)

That 350 width was added to keep it in 2 lines most of the cases. If you need you can try without `!important`.  

Its optional